docs/how-to-guides/renkulab/external-storage.rst

```diff
@@ -0,0 +1,206 @@
+.. _external_storage:
+
+External Storage in Renku Sessions
+==================================
+
+Renku supports accessing data from external storage services like
+S3, Azure Blob, WebDAV, etc. in interactive sessions. Access to external 
+storage can be configured via project settings. 
+
+In the sections below we describe how to add and manage storage configuration to your projects
+and highlight some current limitations:
+
+.. contents::
+  :local: 
+
+.. note::
+
+  Note that this feature is optional; it is enabled at
+  `renkulab.io <http://renkulab.io>`_  but other Renku deployments might not
+  support it. Contact your administrator for more details.
+
+.. warning::
+
+  We support both Read-only and Read/Write access modes.
+  Please note that explicitly marking an attached storage as Read-only is
+  **strongly** encouraged every time you don't have write permissions. This
+  prevents unexpected error messages when trying to access the storage.
+
+Adding external storage to a project
+------------------------------------
+
+Currently, we support configuring external storage access per project.
+Under the project's  ``Settings`` tab, you can find the ``Cloud Storage``
+section. From there, you can click on the ``Add Cloud Storage`` button and follow the
+instructions to go through the guided procedure.
+
+.. image:: ../../_static/images/cloud_storage_1.png
+  :align: center
+  :alt: Cloud Storage section in the project settings
+
+Advanced Mode
```

Comment:
should this be somewhere towards the end? a bit odd to have the section for advanced users be toward the top